pom.xml

```diff
@@ -157,14 +174,14 @@
         <dependency>
             <groupId>org.apache.felix</groupId>
             <artifactId>org.apache.felix.framework</artifactId>
-            <version>7.0.1</version>
-            <scope>provided</scope>
+            <version>7.0.3</version>
+            <scope>compile</scope>
```

Comment:
this is huge and not necessary in all cases, isn't there the possibility to make the Mojo run without it? It is not really expected that a JSP Compiler Mojo always need this dependency?

Reply:
I guess you could force people to add it manually - however, I find that more unexpected than having a bit more download. Furthermore, the difference is 26mb dependencies to 23mb dependencies. Not sure it is worthwhile to make users jump through hoops just to save them some 3mb download (especially given the amount of downloading going on anyways). 
Anyways, I'd say this fixes the bug for now - if we want to change it we can do it in an improvement issue.